src/lib/components/pill.svelte

```diff
@@ -50,7 +50,7 @@
   }
 
   .lightBlue.active {
-    @apply border-2  bg-blue-50 text-primary;
+    @apply border-2 border-indigo-500 bg-indigo-500;
```

Comment:
Let's just use the existing `interactive` color here

```suggestion
    @apply border-2 border-interactive bg-interactive;
```

Reply:
@laurakwhit The interactive color  was what I thought should be used too, but it shows the indigo on light mode and then it goes to bg-transparent with a white border on dark mode. I also tried brand, but the color was too dark.

Reply:
Looks like you were able to use interactive in your PR and it worked fine.... Not sure if this was a me problem?